test/jsx/attr.civet

```diff
@@ -107,6 +107,22 @@ describe "braced JSX attributes", ->
     <Component x={{b:a.b, c:a.c}} />
   """
 
+  testCase """
+    glob with complex left-hand side
+    ---
+    <Component {f()?.{nx,ny}}>
+    ---
+    let ref;<Component {...{nx:(ref = f())?.nx,ny:ref?.ny}} />
+  """
+
+  testCase """
+    glob with complex left-hand side and more
+    ---
+    <Component {a, f()?.{nx,ny}, b}>
+    ---
+    let ref;<Component a={a} b={b} {...{ nx:(ref = f())?.nx,ny:ref?.ny,}} />
```

Comment:
We probably want to preserve the order here if possible. I believe it can have an impact if the same key also appears in the spread.